src/redisraft.c

```diff
@@ -418,7 +418,11 @@ static RRStatus handleSharding(RedisRaftCtx *rr,
 {
     int slot;
 
-    if (computeHashSlotOrReplyError(rr, ctx, cmds, &slot) != RR_OK) {
+    if (computeHashSlot(rr, cmds, &slot) != RR_OK) {
+        if (ctx) {
```

Comment:
At this point, we always have a `ctx`, don't we?

Reply:
I guess I was thinking that it may be called by a follower, though perhaps thats impossible